t3/main.py

```diff
@@ -739,63 +726,28 @@ def determine_species_based_on_sa(self) -> List[int]:
 
         Returns:
             List[int]: Entries are T3 species indices of species determined to be calculated based on SA.
-
-        Adapters notes:
-            - This method should remain and still return a list of indices.
-            - Now we don't need to check whether the observables themselves should be refined,
-              it's done in determine_species_to_calculate()
-            - Perhaps the get_species_by_label() and get_reaction_by_index() functions
-              should be relocated to the RMG adapter.
-            - Consider passing self.logger to the adapters instead if instantiating a new Logger
-              (there should only be a single instance of it).
-            - Need to pass self.t3['sensitivity'] to the adapters instead of ``arguments``.
         """
         species_keys, pdep_rxns_to_explore = list(), list()
         if not os.path.isdir(self.paths['SA solver']):
             self.logger.error("Could not find the path to RMG's SA solver output folder.\n"
                               "Not performing refinement based on sensitivity analysis!")
             return species_keys
 
-        sa_files = list()
-        for file_ in os.listdir(self.paths['SA solver']):
-            if 'sensitivity' in file_ and file_.endswith(".csv"):
-                sa_files.append(file_)
-
-        for sa_file in sa_files:
-            # iterate through all SA .csv files in the solver folder
-            df = pd.read_csv(os.path.join(self.paths['SA solver'], sa_file))
-            sa_dict = {'rxn': dict(), 'spc': dict()}
-            for header in df.columns:
-                # iterate through all headers in the SA .csv file, but skip the `Time (s)` column
-                sa_type = None
-                if 'dln[k' in header and self.t3['sensitivity']['top_SA_reactions']:
-                    sa_type = 'rxn'
-                elif 'dG' in header and self.t3['sensitivity']['top_SA_species']:
-                    sa_type = 'spc'
-                if sa_type is not None:
-                    # proceed only if we care about this column
+        # create new dictionary that stores the absolute maximum value from each SA
+        sa_dict_max = {'rxn': dict(), 'spc': dict()}
```

Comment:
`sa_dict_max` is not an attribute of the T3 class since it is just an intermediate step to calculating `species_keys`. For this reason, I had thought it was okay to use this code since it's not used outside of this method

Reply:
right, it's not an attribute, but maybe we could still use consistent syntax?

Reply:
Sure thing. I updated the keys to `kinetics` and `thermo` to be consistent with the keys in `sa_dict`